### PR TITLE
Duplicates merged

### PR DESCRIPTION
**Pull request by loma-one**
Duplicates merged & emoji enabled/disabled

### CODE REVIEW

File: unicode_smilies/unicode_smilies.php [hunk 1]
```diff
@@ -22,12 +22,13 @@ function unicode_smilies_smilies(&$a,&$b) {
 	Smilies::add($b, ':-D', '&#x1F601;');
 	Smilies::add($b, ':D', '&#x1F601;');
 	Smilies::add($b, ';-)', '&#x1F609;');
-	// Smilies::add($b, ';)', '&#x1F609;'); // Deactivated since this leads to disturbed html entities
+	Smilies::add($b, ';)', '&#x1F609;');
```
Comment (annando) [inline unicode_smilies/unicode_smilies.php:25]: I guess that we should keep this deactivated. See the reason in the comment.

Comment (loma-one) [reply unicode_smilies/unicode_smilies.php:25]: Thank you

**Comment by loma-one** [comment]
@MrPetovan This is not a duplicate. Here was cleaned up and duplicate emoji merged, or activated / deactivated.

**Comment by MrPetovan** [comment]
His PR was automatically closed because I deleted the RC branch after the release. Please rebase your branch on too of `develop` and open a new PR for it.

**Pull request closed without merging**